### ISSUE TITLE
Which d4utils version to use?

### ISSUE THREAD

**Issue by javiercguard** [38/d4-format#35]
Hi,
I was interested in using d4utils on combination with modepth, but I am not sure if I should build for source, or use the binary in the release, which is a year old, while the master branch seems to be updated recently. If the binary in the release is usable, I suppose it could be good for a bioconda package, too.

This may be related to issue (suggestion) #11, which asks for automatic binaries deployment.

Thanks for your attention.

**Comment by 38** [comment]
Yep, we are going to publish a newer version after the recent work completes. At this time, you probably can use the the published version - as the file specification doesn't change. 

@arq5x It's possible for automatically build and upload the artifact with github action.
